test/mysql.test.js

```diff
@@ -41,13 +41,22 @@ describe('mysql', function() {
       title: {type: String, length: 255, index: true},
       content: {type: String},
     });
-
+    PostWithNumId = db.define('PostWithNumId', {
+      id: {type: Number, id: true},
```

Comment:
I might wrong, `this.isNullable(prop)` still returns false if we don't specify `null:true` right?
I remember yesterday you mentioned MYSQL allows id to be null, so I am not sure do you want to test id nullable here or id not nullable..